raiden-ts/src/channels/epics.ts

```diff
@@ -271,41 +271,53 @@ function scanRegistryTokenNetworks({
     filter(([, tokenNetwork]) => !!tokenNetwork),
     toArray(),
     mergeMap((logs) => {
-      if (!logs.length) return EMPTY;
-      const firstBlock = logs[0][2].blockNumber;
-      const tokenNetworks = new Map<string, [token: Address, event: Event]>(
-        logs.map(([token, tokenNetwork, event]) => [tokenNetwork, [token, event]]),
-      );
-      const allTokenNetworkAddrs = Array.from(tokenNetworks.keys());
-      const aTokenNetworkContract = getTokenNetworkContract(allTokenNetworkAddrs[0] as Address);
-      const ChannelOpenedTopic = Interface.getEventTopic(
-        aTokenNetworkContract.interface.getEvent('ChannelOpened'),
+      const alwaysMonitored$: Observable<tokenMonitored> = from(
+        logs
+          .splice(0, 2)
```

Comment:
why two? For the purpose of remembering the sync one would be enough, so shouldn't we just do this or is this done with thought of current mainnet - so that both tokens there would be synced?

Reply:
Yeah, this is a little usecase-centric, so 2 is just arbitrarily chosen. Mainnet is the most obvious, but on Goerli, we also make more extensive usage of the first 2 registered TNs. Since this is a temporary code, and also on a fallback case (the user can `monitorToken` explicitly and if they get a (past or new) channel on one of the other TNs, they'll be monitored anyways), I don't think it's an issue to have this default expected case hardcoded here.